tests/test_x509_ext.py

```diff
@@ -1400,18 +1434,18 @@ def test_not_name(self):
 
     def test_repr(self):
         name = x509.Name([x509.NameAttribute(NameOID.COMMON_NAME, u'value1')])
-        gn = x509.DirectoryName(x509.Name([name]))
+        gn = x509.DirectoryName(name)
```

Comment:
Great catch, but would you mind putting these fixes in as a separate PR and removing them from this one? Sorry to make you do some extra work :(